<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostTrackingWindow</class>
 <widget class="QWidget" name="PostTrackingWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="PwGisWidget" name="pwgiswidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLineEdit" name="lineEdit_lon">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Время жизни (сек.)</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QSlider" name="horizontalSlider_course">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="sliderPosition">
        <number>0</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QPushButton" name="btnStopTracking">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Остановить</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_setPost">
       <property name="text">
        <string>Установить</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_lat">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QPushButton" name="btnOpenMap">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Открыть карту</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLCDNumber" name="lcdNumber_course">
       <property name="numDigits">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Задать курс поста:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Lat:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Lon:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Стиль поста:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QSpinBox" name="onLifeTime">
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QSpinBox" name="onTailLength">
       <property name="maximum">
        <number>40000</number>
       </property>
       <property name="singleStep">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QPushButton" name="btnStartTracking">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Запустить</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Длина хвоста (км.)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
